### PR TITLE
Add c8 compatibility 

### PR DESCRIPTION
**Pull request by Vancelot11**
DO NOT BUMP production_centos6 or phpbb env

### CODE REVIEW

**Comment by Vancelot11** [comment]
[osl-acme](https://trello.com/c/jiuJu7JU/1590-osl-acme)

File: metadata.rb [hunk 1]
```diff
@@ -13,5 +13,6 @@
 depends          'git'
 depends          'resolver'
 
-supports         'centos', '~> 6.0'
+# TODO: DO NOT BUMP ON production_centos6 and phpbb until migration
```
Comment (ramereth) [inline metadata.rb]: Let's not have this comment here.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -11,9 +11,15 @@
   version: '6',
 }.freeze
 
+CENTOS_8 = {
+  platform: 'centos',
+  version: '8',
+}.freeze
+
 ALL_PLATFORMS = [
   CENTOS_6,
```
Comment (ramereth) [inline spec/spec_helper.rb]: You should remove this and any CentOS 6 related tests.

**Comment by ramereth** [comment]
@Vancelot11 you need to fix this logic since everything will be systemd w/o CentOS 6 https://github.com/osuosl-cookbooks/osl-acme/blob/vancelot/c8/recipes/server.rb#L72-L86

**Comment by Vancelot11** [comment]
@ramereth should I just bump production,openstack_rocky, and workstation?

**Comment by Vancelot11** [comment]
!bump minor production,openstack_rocky,workstation

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v3.1.0.  Have a nice day!

**Pull request merged**